python/ray/dashboard/node_stats.py

```diff
@@ -16,6 +16,59 @@
 
 logger = logging.getLogger(__name__)
 
+PYCLASSNAME_RE = re.compile(r"(.+?)\(")
+
+
+def group_actors_by_python_class(actors):
+    groups = defaultdict(list)
+    for actor in actors.values():
+        actor_title = actor.get("actorTitle")
+        if not actor_title:
+            groups["Unknown Class"].append(actor)
+        else:
+            match = PYCLASSNAME_RE.search(actor_title)
+            if match:
+                # Catches case of actorTitle like
+                # Foo(bar, baz, [1,2,3]) -> Foo
+                class_name = match.groups()[0]
+                groups[class_name].append(actor)
+            else:
+                # Catches case of e.g. just Foo
+                # in case of actor task
+                groups[actor_title].append(actor)
+    return groups
+
+
+def get_actor_group_stats(group):
```

Comment:
Why don't we move them into a `NodeStats` class? They look more like a private method in my opinion because it doesn't look like we can reuse these methods.

Reply:
It could be a static method, you're right. I left it as a function because it doesn't use any of the state of NodeStats, but it could just as well be a static method.

Reply:
Personally I don't see that much of a difference between the two, but I'll put an underscore in front of these names regardless to indicate they shouldn't be used outside the module.